bridge/opentracing/bridge.go

```diff
@@ -130,6 +130,14 @@ func (s *bridgeSpan) SetOperationName(operationName string) ot.Span {
 	return s
 }
 
+// SetTag method adds a tag to the span.
+//
+// Note about the following value conversions:
+// - uint -> int
```

Comment:
From the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/9e6a3de19cb3db983b2e4d56b37ec6313fd4ffc6/specification/common/common.md#attributes):

> A primitive type: string, boolean, double precision floating point (IEEE 754-1985) or signed 64 bit integer.

IIUC, We can only have these types: `string`, `bool`, `float64` and `int64`. Should we also remove the `int` type?

Reply:
Should just make func description clearly because we take `int` but still convert it to `int64`:
https://github.com/dstdfx/opentelemetry-go/blob/6e3f64dfd389b008578ce5ff192f8166bf95eaff/label/key.go#L73